### PR TITLE
Display editors and reviewers on reviews page

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

File: src/reviews/reviews-editors-and-reviewers.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const editorsAndReviewers = (): string => `
+<section class="editors-and-reviewers">
+  <h2 class="editors-and-reviewers__header" id="editors-and-reviewers">Editors and reviewers</h2>
+  <div class="editors-and-reviewers__person">
+    <div class="editors-and-reviewers__person__role">Reviewing Editor</div>
```
Comment (scottaubrey) [inline src/reviews/reviews-editors-and-reviewers.ts:5]: This doesn't feel quite right to me - the role is related to the name of the person, and together are the headers. 

I'm thinking either change the h3 to div and use styles to give it more visual prominence or role moved into the header element, then styled to look like it does in the design by wrapping the different elements in a span or something?

I think one way or another, we should also put this into a `<ul>`, as they are semantically a list.

I'd get someone else to suggest solutions though, It's not my area of expertise.

File: src/reviews/reviews.ts [hunk 2]
```diff
@@ -18,8 +20,9 @@ const wrapWithHtml = (reviews: string, doi: string, noHeader: boolean): string =
 
 export const generateReviewPage = (article: EnhancedArticle, noHeader: boolean): string => {
   if (article.reviews.length === 0) {
-    return wrapWithHtml('<li class="review-list__item"><article class="review-list-content">No reviews found</article></li>', article.doi, noHeader);
+    return wrapWithHtml(editorsAndReviewers(), '<li class="review-list__item"><article class="review-list-content">No reviews found</article></li>', article.doi, noHeader);
```
Comment (scottaubrey) [inline src/reviews/reviews.ts:23]: If there are no reviews, are there any editors and reviewers? If not, probably just pass a blank string?

Comment (acollings) [reply src/reviews/reviews.ts:23]: No, I'd say not. Once the Reviewed Preprint (preprint + reviews) is available, displaying editor name(s) is fine, but we're not going to display reviewer names (for eLife ones at any rate). We wouldn't want editor or reviewer names displayed before reviews are available. Hope that helps!

Comment (nlisgo) [reply src/reviews/reviews.ts:23]: Thanks @acollings. Maybe we can base the decision to display the editors and reviewers on their inclusion in the docmap file which means we don't need to worry about the logic in this application.

I have added some commentary on this to the parent issue: https://github.com/elifesciences/enhanced-preprints/issues/119#issuecomment-1165283167

**Review by scottaubrey — APPROVED** [review]
I've brought the branch right up to date with master changes, and I'm happy either way with displaying the static HTML for now. 

The last remaining issues are around spacing between elements (which has only appeared as we now have multiple elements), but I've created #169 to cover that. Happy to merge this now.

**Comment by nlisgo** [comment]
My issue with this is the same as yours. Ideally we would be passing a parameter to `editorsAndReviewers()`, a parameter that we don't use currently but can be used at some point to determine whether there are editors and reviewers for this preprint. The coding standards would fail if we have a parameter that we don't need currently.

**Pull request merged**